harness/determined/exec/harness.py

```diff
@@ -163,7 +163,15 @@ def main() -> None:
     container_id = os.environ["DET_CONTAINER_ID"]
     hparams = simplejson.loads(os.environ["DET_HPARAMS"])
     initial_work = workload.Workload.from_json(simplejson.loads(os.environ["DET_INITIAL_WORKLOAD"]))
-    latest_checkpoint = simplejson.loads(os.environ["DET_LATEST_CHECKPOINT"])
+
+    # Load latest checkpoint from file to handle large data format
+    with open("checkpoint.txt", "w") as f:
+        f.write(os.environ["DET_LATEST_CHECKPOINT"])
```

Comment:
wait, should this file be packed into the container by the master instead of by the harness code?